sysbuild/Kconfig.suit

```diff
@@ -0,0 +1,94 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "SUIT"
+
+menuconfig SUIT_ENVELOPE
```

Comment:
Why do we need this here if it is already defined in the suit-generator repository? Do you want this to replace the Kconfig from the suit-generator?

Reply:
The Kconfig in suit-generator repository is removed in the PR from the manifest.